pkgs/development/tools/misc/pwndbg/default.nix

```diff
@@ -40,7 +41,8 @@ in stdenv.mkDerivation rec {
     chmod +x $out/share/pwndbg/gdbinit.py
     makeWrapper ${gdb}/bin/gdb $out/bin/pwndbg \
       --add-flags "-q -x $out/share/pwndbg/gdbinit.py" \
-      --set NIX_PYTHONPATH ${pythonPath}
+      --set NIX_PYTHONPATH ${pythonPath} \
+      --prefix NIX_DEBUG_INFO_DIRS : ${glibc.debug}/lib/debug
```

Comment:
the debug output is only created on https://github.com/NixOS/nixpkgs/blob/385ea92713cc98967be6c2bd47c951da3e07bf45/pkgs/stdenv/generic/make-derivation.nix#L173

make this conditional with `lib.elem "debug" glibc.outputs` and it'll work